src/dmd/errors.d

```diff
@@ -21,7 +21,7 @@ import dmd.root.outbuffer;
 import dmd.root.rmem;
 import dmd.console;
 
-/**********************
```

Comment:
It's not really necessary to do this. The number of *'s is not consistent anywhere in the source code, and it hardly matters.